src/umpire/util/error.hpp

```diff
@@ -10,7 +10,7 @@
 #include <stdexcept>
 #include <string>
 
-#include "umpire/fmt/fmt.hpp"
+#include "fmt/format.h"
```

Comment:
Are we OK with exposing fmt in a header?

Reply:
Looks like it's not even used in this file. So now it shouldn't bleed out of Umpire's headers.

Reply:
Oh, but it will propagate into other headers, now.

Reply:
It would be nice to avoid this, but I think that's better left as a future exercise.